### PR TITLE
New package: AlignTechnology,Inc.InvisalignOutcomeSimulator version 4.2.1.75

### PR DESCRIPTION
**Pull request by DavidHGAK**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138186)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335226 2024-02-06 18:19:24.728 [CLI ] ShellExecute installer failed: 2147753984

(Automated response - build 707.)

**Comment by stephengillie** [comment]
Hi @DavidHGAK,

This package didn't fully install when using the silent install switch, but did when running the attended installer - it required an EULA agreement that didn't have a counterpart installer swtich:
image

I'm wondering if there's a missing installer switch to accept the license. I searched but couldn't find one. I'm guessing that the error code we're seeing is the license agreement failure error. 

This presents another issue in that we've [recently had to remove](https://github.com/microsoft/winget-pkgs/issues/131328) several packages that have EULA or similar agreements in the installer. 

Let me know what you think.

File: manifests/a/AlignTechnology,Inc/InvisalignOutcomeSimulator/4.2.1.75/AlignTechnology,Inc.InvisalignOutcomeSimulator.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: AlignTechnology,Inc.InvisalignOutcomeSimulator
```
Comment (Trenly) [inline manifests/a/AlignTechnology,Inc/InvisalignOutcomeSimulator/4.2.1.75/AlignTechnology,Inc.InvisalignOutcomeSimulator.installer.yaml:4]:
```suggestion
PackageIdentifier: AlignTechnology.InvisalignOutcomeSimulator
```
After changing this, the folder and files will need to be renamed

File: manifests/a/AlignTechnology,Inc/InvisalignOutcomeSimulator/4.2.1.75/AlignTechnology,Inc.InvisalignOutcomeSimulator.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: AlignTechnology,Inc.InvisalignOutcomeSimulator
```
Comment (Trenly) [inline manifests/a/AlignTechnology,Inc/InvisalignOutcomeSimulator/4.2.1.75/AlignTechnology,Inc.InvisalignOutcomeSimulator.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: AlignTechnology.InvisalignOutcomeSimulator
```
After changing this, the folder and files will need to be renamed

File: manifests/a/AlignTechnology,Inc/InvisalignOutcomeSimulator/4.2.1.75/AlignTechnology,Inc.InvisalignOutcomeSimulator.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: AlignTechnology,Inc.InvisalignOutcomeSimulator
```
Comment (Trenly) [inline manifests/a/AlignTechnology,Inc/InvisalignOutcomeSimulator/4.2.1.75/AlignTechnology,Inc.InvisalignOutcomeSimulator.yaml:4]:
```suggestion
PackageIdentifier: AlignTechnology.InvisalignOutcomeSimulator
```
After changing this, the folder and files will need to be renamed

**Review by Trenly — CHANGES_REQUESTED** [review]
It seems this application may require an interactive install

**Pull request closed without merging**